web/pandas/config.yml

```
@@ -34,7 +34,7 @@ navbar:
     - name: "Citing and logo"
       target: /about/citing.html
   - name: "Getting started"
-    target: /getting_started.html
+    target: docs/getting_started/index.html
```

Let's add the leading slash to make it consistent.

```suggestion
    target: /docs/getting_started/index.html
```

---

> * [x]  Have the navigation bar point to `/docs/getting_started/index.html`
> * [x]  Delete [the old page](https://github.com/pandas-dev/pandas/blob/master/web/pandas/getting_started.md)
> * [x]  [Add a redirect](https://github.com/pandas-dev/pandas/blob/master/doc/redirects.csv)

@afeld 
Requested amendments have been followed through...